rails/app/javascript/components/StoryList.jsx

```diff
@@ -63,7 +63,8 @@ class StoryList extends Component {
             style={style}
           >
             <div className="speakers">
-              <img src={story.speaker.picture_url} alt={story.speaker.name} title={story.speaker.name}/>
+              <img src={story.speaker.picture_url}/>
```

Comment:
We might want to keep the alt and title properties here for accessibility reasons. What do you think?

Reply:
I checked it out with it and without it. I really didn't see the difference but I have no problem with adding that back in.

Reply:
I think when you use screen readers, that’s how it knows how to describe the images. I haven’t tested it out myself though.

Reply:
oh ok I understand what you are saying. Adding that back in now.